packages/password-field/src/vaadin-password-field.js

```diff
@@ -174,6 +175,9 @@ export class PasswordField extends SlotStylesMixin(TextField) {
     if (this.inputElement) {
       this.inputElement.autocapitalize = 'off';
     }
+
+    this._tooltipController = new TooltipController(this);
```

Comment:
We shouldn't need to create a controller here, as it's inherited from `TextField`.

Reply:
Done.